nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -236,7 +242,31 @@ in
 
     systemd.packages = [ cfg.package ];
 
-    systemd.services.ipfs = {
+    systemd.services.ipfs =
```

Comment:
If there's another way to do this, it'd be great. I just haven't been able to find one, this PR is more of an idea that I had and implemented poorly. I would be very happy if someone could formulate a better way of doing this.

Reply:
I mentioned below that putting each list item into its own `systemd` service is an alternative. I'm not sure if it is the right alternative here or not, but it would avoid the restart.